### DISCUSSION TITLE
Firefox can't use notify-send

### DISCUSSION THREAD

**Discussion by ibahnasy** [netblue30/firejail#4401] [Q&A] [answered]
Firefox is unable to use notify-send when ran with firefjail, instead it show a popup window for notifications.
What do I need to allow in firejail to let it use notify-send as usual?

**Comment by rusty-snake** [comment, marked as answer]
Does firefox use `notify-send`? Would be new to me. AFAICTY firefox talks directly to `org.freedesktop.Notifications` without the indirection of starting a program (maybe it uses libnotify but not notify-send). Because for me it works and I've a `private-bin` without `notify-send`.

Anyway since `private-bin` isn't enabled by default all you need it likely

https://github.com/netblue30/firejail/blob/110a74f094abcb4f2763d76e204fb3c9743fa9a1/etc/profile-a-l/firefox.profile#L51-L52
